scripts/NuGetUtilities.psm1

```diff
@@ -0,0 +1,155 @@
+<#
+.SYNOPSIS
+    NuGet utility functions.
```

Comment:
>functions [](start = 18, length = 9)

Much of the code in this module was just extracted from `ScriptUtilities.psm1`. The new code starts on Line 97 with the function `Get-NuGetApiKey`.